### ISSUE TITLE
Clarification about main aim of the powershell-community blog space

### ISSUE THREAD

**Issue by ALIENQuake** [PowerShell/Community-Blog#30]
Hi,

After discussion with @doctordns I would like to request clarification about the main goal of the powershell-community blog space. The general term "to provide a blogging platform" doesn't really explain the aim of this platform.

Is there even an aim? Like teaching newcomers how to use PowerShell? Cover advanced PowerShell topics? Or post anything PS-related?

My suggestion regarding this matter is that this platform has a great opportunity to have a general goal of being a great source for basic "how-to" examples and at the same time, have additional content to cover the widest audience of systems/solutions without necessarily focusing on code/solution simplicity.

In order to encourage authors to search for the simplest solution and accept PR's that covers those, how about having general guidance as of "If they exist, provide a simples way of achieving the goal for newcomers, then follow your way."?

**Comment by doctordns** [comment]
A great question! And I agree with the points you make.

Looking back to the early days of PowerShell, the various scripting guys, and their blogs, were an inspiration to all. There were great posts showing how to do good things in PowerShell and other scripting tools. The articles were well written and told a great story. Sadly, with the passing of time and Ed Wilson's retirement, the blog had grown stale. I wanted to invigorate it. Which is how we get to where we are today. 

I believe this blog should be a collection of practical, simple, and well-written articles that highlight what you can do with PowerShell, particularly PowerShell Core/PowerShell 7.  Many of the earlier blog articles are, as my first few contributions hopefully have shown, are still relevant today (once brought up to date). I also want this blog to be search engine friendly as well.  Thus my vision, at least, was for a continuation of that tradition - great articles that help folks. 

As an aside, I was looking for a simple way to change the volume label and drive letter for USB drive and Google directed me to a great page, but the page had a typo that meant cut/pasting the code failed (the curly vs straight quote issue!). That prompted me to ask the question as to what should be done. I believe that creating a _new_ blog and opening it up to the community for contributions to be an outstanding idea. SO long as the articles are high quality and focused. 

One thing I see over at Spiceworks where I moderate [the Spiceworks PowerShell forum](https://community.spiceworks.com/programming/powershell) is that there continue to be new folks getting into PowerShell. From that, I feel we should be aiming this at the 200-300 level. The blog should definitely have some deep dives but should focus on mainstream topics.  

In terms of the writing of articles, I want to see ones that are clear and focus on a specific point. they should have good PowerShell code and be easy to cut and paste. The articles need to be written in good technical American English. So that means no colour, no organise, etc. It also means avoiding passive voice, future tense, and split infinitives.  I wrote a post some time back on [my top 10 tips for technical writing](https://tfl09.blogspot.com/2016/09/my-ten-tips-for-technical-writing.html). While we do not want to get too dogmatic, those tips continue to be important today in terms of this blog. I am working with the powers that be to create some more specific guidance as to the standards required for posts here. 

In thinking this through, we came to the view that an all-new blog provided a better approach. I think this is very much the right idea although I am keen to ensure as much of the older content can get carried forward, altered where appropriate. I hope the community agrees with that decision. 

So, in summary, this blog is very much a work in progress and we learn as we go along. A key question is what can we do more of,  or do differently, so as to benefit the community?

**Comment by sdwheeler** [comment]
See the README and https://devblogs.microsoft.com/powershell-community/about
